benchmarks-api/src/main/java/io/scalecube/benchmarks/BenchmarksState.java

```diff
@@ -51,6 +52,9 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(BenchmarksState.class);
 
+  private static final String MDC_BENCHMARK_DIR = "benchmarkDir";
+  private static final String MDC_BENCHMARK_NAME = "benchmarkName";
```

Comment:
We dont need `MDC_BENCHMARK_NAME` since `MDC_BENCHMARK_DIR` is already uniquily defines path to log directory